install/kubernetes/helm/istio/values.yaml

```diff
@@ -157,7 +157,9 @@ gateways:
     loadBalancerIP: ""
     type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
     ports:
+      ## You can add custom gateway ports
```

Comment:
@rshriram can you help check if this PR is good?

Reply:
? This is empty.

Reply:
This is just a comment, and you may see I also add a new item `targetPort: 80` below.

Reply:
Isn’t target port 80 by default?

Reply:
Yes, targetport is 80 by default, but here we need to distinguish it between `port` and `targetport`, as if I did not specify `targetport:80` but only `port:80`, both `port` and `targetport` will be 80, this caused me cannot customize the service port.